wallet/package.json

```diff
@@ -4,58 +4,117 @@
     "font-awesome": "4.7.0",
+    "fs-extra": "^9.0.1",
     "general-number": "^1.0.1",
+    "html-webpack-plugin": "4.5.0",
     "idb": "^7.0.0",
+    "identity-obj-proxy": "3.0.0",
+    "jest": "26.6.0",
+    "jest-circus": "26.6.0",
+    "jest-resolve": "26.6.0",
+    "jest-watch-typeahead": "0.6.1",
     "js-sha3": "^0.8.0",
     "keccak": "^3.0.2",
     "lottie-react": "^2.2.1",
+    "mini-css-extract-plugin": "0.11.3",
+    "optimize-css-assets-webpack-plugin": "5.0.4",
+    "pnp-webpack-plugin": "1.6.4",
+    "postcss-flexbugs-fixes": "4.2.1",
+    "postcss-loader": "3.0.0",
+    "postcss-normalize": "8.0.1",
+    "postcss-preset-env": "6.7.0",
+    "postcss-safe-parser": "5.0.2",
+    "prompts": "2.4.0",
     "prop-types": "^15.7.2",
-    "qrcode.react": "^3.0.0",
+    "qrcode.react": "^2.0.0",
     "queue": "^6.0.2",
     "react": "^17.0.2",
+    "react-app-polyfill": "^2.0.0",
     "react-bootstrap": "^2.1.1",
     "react-copy-to-clipboard": "^5.0.4",
+    "react-dev-utils": "^11.0.3",
     "react-dom": "^17.0.2",
     "react-icons": "^4.3.1",
     "react-redux": "7.2.5",
+    "react-refresh": "^0.8.3",
     "react-responsive": "^9.0.0-beta.6",
     "react-router-dom": "5.3.0",
     "react-to-webcomponent": "^1.5.1",
     "react-toastify": "^8.2.0",
     "redux": "4.1.1",
     "redux-logger": "^3.0.6",
     "redux-thunk": "^2.3.0",
+    "resolve": "1.18.1",
+    "resolve-url-loader": "^3.1.2",
     "safe-buffer": "^5.2.1",
     "sass": "^1.49.7",
-    "typescript": "4.4.4",
+    "sass-loader": "^10.0.5",
+    "semver": "7.3.2",
+    "style-loader": "1.3.0",
+    "styled-components": "^5.3.5",
+    "systemjs-webpack-interop": "^2.3.7",
+    "terser-webpack-plugin": "4.2.3",
+    "ts-pnp": "1.2.0",
+    "typescript": "^4.5.5",
+    "url-loader": "4.1.1",
     "wasm-loader": "^1.3.0",
     "web-vitals": "^2.1.0",
     "web3": "^1.6.0",
-    "webpack-manifest-plugin": "^4.1.1",
+    "webpack": "4.44.2",
+    "webpack-dev-server": "3.11.1",
+    "webpack-manifest-plugin": "2.2.0",
+    "workbox-webpack-plugin": "5.1.4",
     "zokrates-js": "^1.0.39"
   },
   "scripts": {
-    "start": "LOCAL_PROPOSER=false USE_STUBS=true craco start",
+    "start": "LOCAL_PROPOSER=true PUBLIC_URL='/nightfall/' USE_STUBS=true node scripts/start.js",
     "start:docker": "REACT_APP_ENVIRONMENT=Docker PORT=3010 craco start",
     "start:ropsten": "REACT_APP_ENVIRONMENT=Ropsten craco start",
-    "build": "LOCAL_PROPOSER=false USE_STUBS=true craco build",
+    "build": "LOCAL_PROPOSER=false craco build",
+    "build:integration": "LOCAL_PROPOSER=false USE_STUBS=false MODE=testnet PUBLIC_URL='/nightfall/' node scripts/build.js",
     "deploy": "npm run build && aws s3 sync build s3://pnf-dev-browser --cache-control max-age=172800 --delete && aws configure set preview.cloudfront true && aws cloudfront create-invalidation --distribution-id ENV3X13MLR5YT --paths \"/*\"",
     "test": "craco test --env=jsdom",
-    "eject": "craco eject",
+    "test:jest": "jest",
```

Comment:
this test command does not work for me
<img width="1728" alt="image" src="https://user-images.githubusercontent.com/9293647/164411551-f1ff239d-04f8-4c47-ad77-301138bfe3b5.png">